kubernetes/template-generator/cli.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+from template_generator import *
+
+
+LEAPP_BASEPATH = "/var/lib/leapp/macrocontainers"
+
+
+def parse_params():
+    def _parse_port(arg):
+        docker_port, sep, container_port = arg.partition(":")
+        if not container_port:
+            container_port = docker_port
+
+        return int(docker_port), int(container_port)
+
+    def _parse_ip(arg):
+        ## TODO add proper parser
+        return arg
+
+    parser = argparse.ArgumentParser(description='Generate kubernetes tempates.')
+    subparser = parser.add_subparsers(dest="generator")
+
+    multi = subparser.add_parser("multivolume", help="test")
+    macro = subparser.add_parser("macroimage", help="test2")
+
+    #selectors = []
+    mandatory_groups = []
+
+    for p in multi, macro:
+        p.add_argument("--tcp", "-t", type=_parse_port, nargs="*", help="Exposed TCP ports")
+        p.add_argument("--ip", "-i", type=_parse_ip, nargs="+", help="IP address on which the ports will be exposed")
+        p.add_argument("--dest", "-e", default=os.getcwd(), help="Generate service template (default: true)")
+
+        mandatory = p.add_argument_group("Mandatory arguments")
+        mandatory.add_argument("--container-name", "-c", help="A name of macrocontainer", required=True)
+        mandatory_groups.extend([mandatory])
+
+        #selector = p.add_argument_group("Template selectors (optional)")
+        #selector.add_argument("--pod", "-p", action="store_true", default=None, help="Generate pod template (default: true)")
```

Comment:
cli.py:42:80: E501 line too long (126 ) - I believe we agreed on 120 line length?